service/controller/v3/resource_set.go

```diff
@@ -169,6 +192,7 @@ func NewResourceSet(config ResourceSetConfig) (*controller.ResourceSet, error) {
 		networkConfigResource,
 		namespaceResource,
 		legacyResource,
+		flanneldResource,
```

Comment:
We are finally using the `flanneld` resource we implemented like a year ago and never really used. 🎉

Reply:
I think the initial idea was to GC everything with the namespace. But that apparently doesn't work.